cli/cli.go

```diff
@@ -92,6 +96,8 @@ func before(c *cli.Context) error {
 		configs.IsPullRequestMode = true
 	}
 
+	configs.BitriseVersionStr = c.App.Version
```

Comment:
make it `Freezable` - see example in https://github.com/bitrise-tools/bitrise-machine/blob/master/cli/cli.go#L36